src/firefly/js/visualize/ui/ImageSelectPanel.jsx

```diff
@@ -82,7 +81,11 @@ export function computeCurrentCatalogId( fields, catalogId = IRSA ) {
 // this version is made to add a new plot to an existing viewer if no plot is found
 // or replace the plot from the active viewer
 
-const PlotSelectMode = new Enum(['AddNewPlot', 'ReplacePlot', 'ReplaceOrCreate']);
+const [REPLACE, CREATE, CREATE3COLOR] = [0x100, 0x010, 0x001];
```

Comment:
I have never thought of using this syntax like this before.  It is a great way to defining related constants!